ParisBNF/abb/BNFabb87.xml

```diff
@@ -0,0 +1,214 @@
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno>BnF Éthiopien d'Abbadie 87</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title type="complete" ref="LIT4078Collectionch">Collection of antiphons</title>
+                            <textLang mainLang="gez"/>
+                            <note>Content description is based on 
+                                <bibl>
+                                    <ptr target="bm:Shelemay1993"/>
+                                    <citedRange unit="page">85</citedRange>
+                                    <citedRange unit="note">43</citedRange>
+                                </bibl>  
+                            </note>
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="1r" to="68v"/>
+                                <title type="complete"><foreign xml:lang="gez">mazmur antiphons</foreign></title>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="69r" to="92v"/>
+                                <title type="complete"><foreign xml:lang="gez">salām and wāzemā antiphons</foreign></title>
+                            </msItem>
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="92v" to="106v"/>
+                                <title type="complete"><foreign xml:lang="gez">wāzemā antiphons</foreign></title>
+                            </msItem>
+                            <msItem xml:id="ms_i1.4">
+                                <locus from="106v" to="120v"/>
+                                <title type="complete"><foreign xml:lang="gez">ʿǝzl antiphons</foreign></title>
+                            </msItem>
+                        <msItem xml:id="ms_i1.5">
+                            <locus from="120v" to="128v"/>
+                            <title type="complete"><foreign xml:lang="gez">za-ʾamlākiya and ʾarbāʿt antiphons</foreign></title>
+                        </msItem>
+                        <msItem xml:id="ms_i1.6">
+                            <locus from="128v" to="140r"/>
+                            <title type="complete"><foreign xml:lang="gez">śalast antiphons</foreign></title>
+                        </msItem>
+                        <msItem xml:id="ms_i1.7">
+                            <locus from="140r" to="148v"/>
+                            <title type="complete"><foreign xml:lang="gez"> ʾaryām antiphons</foreign></title>
```

Comment:
Titles are not capitalised here and above, but capitalised below. I think the titles should be capitalised for the first word.